specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2020-01-01-preview/examples/DeleteNodes_example.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "resRg",
+    "clusterName": "myCluster",
+    "nodeTypeName": "BE",
+    "api-version": "2020-01-01-preview",
+    "parameters": {
+      "nodes": [
+        "BE_0",
+        "BE_3"
```

Comment:
How is deleting lower numbered nodes impact the node numbers?

Reply:
I'm not sure I understand the question. these are the node names to be deleted. if we have be_0, be_1, and be_2 and I remove be_0 then we will have after the operation only be_1 and be_2